### DISCUSSION TITLE
Can't get no-touch-required to work

### DISCUSSION THREAD

**Discussion by knsonier** [PowerShell/Win32-OpenSSH#2030] [Q&A]
I have v9.1.0.0p1-Beta installed on a Windows 10 machine and generated a key with the command:
`ssh-keygen -t ed25519-sk -O no-touch-required -O verify-required -f my-key`

I added the pub key to my authorized_keys file on my Linux server with the no-touch-required option and tried to ssh from Windows with this being the result:
`sign_and_send_pubkey: signing failed for ED25519-SK "my-key": requested feature not supported`

I then tried to use the key from another Linux machine and I can ssh into the server without any problem so I'm a little bit confused. I can generate a working key with no-touch-required on Windows 10 but I can't actually use it? Google hasn't been very helpful at trying to solve my problem so I'm hoping someone here can help. I hope this made sense...

Thank you for your help!

**Comment by PaulHigin** [comment]
+ @anmenaga  Can you comment?

**Comment by bellegarde-c** [comment]
Facing the exact same issue here, I can create the key but I can't use it.

**Comment by beachrunner-au** [comment]
On the server you are trying to connect to (the Linux host), do you have the option "no-touch-required" specified before and on the same line as your public key in ~/.ssh/authorized_keys? sshd's default behaviour is to reject no-touch-required keys.

More info here:
https://wiki.archlinux.org/title/SSH_keys#:~:text=Additionally%2C,authorized_keys

**Reply by knsonier** [reply]
Yes, I had that setting in the authorized_keys file on the same line as the pub key.
